### ISSUE TITLE
Add support for v6 (latest)

### ISSUE THREAD

**Issue by griimick** [catppuccin/codemirror#2]
### Resources
1. [Official Styling docs](https://codemirror.net/examples/styling/#themes)
2. [Sample official theme - One Dark](https://github.com/codemirror/theme-one-dark)

**Comment by leessmin** [comment]
Excuse me, is it available in 6 now

**Comment by BuonOmo** [comment]
If someone wants to take the lead on this, there is already a nice backbone for it in [ahdinosaur/codemirror-theme-catppuccin](https://github.com/ahdinosaur/codemirror-theme-catppuccin).

The only issue left to tackle is that the colors are not matching this repository properly!

**Comment by sgoudham** [comment]
> EDIT: I've done it for a local project and could definitely clean it and make a pull request, but I don't want to maintain the project nor to push it on npm. So ping me if you're taking the project back and I'll do the PR :)

This would be very much appreciated! Feel free to request me for review on the eventual PR.

**Comment by BuonOmo** [comment]
@sgoudham would you also handle the pushing to npm part ?

**Comment by sgoudham** [comment]
Yup, I'll push a release workflow on top of your PR or add it afterwards 👍
